hypothesis_test.ipynb

```diff
@@ -4,7 +4,7 @@
   "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.          'fraud': [True, False, True, True, False, True, False],
</pre><p>Is the <code>fraud</code> output a label here or a prediction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf_common/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>